src/core/Jalasoft.TeamUp.Resumes.Core/Jalasoft.TeamUp.Resumes.Core.csproj

```diff
@@ -6,11 +6,13 @@
 	</PropertyGroup>
 
 	<ItemGroup>
+		<PackageReference Include="Dapper" Version="2.0.90" />
```

Comment:
question, why is dapper and System.Data.SqlClient being referenced at core layer? core layer shouldn't talk sql language